### ISSUE TITLE
Move hasPEPStatus inside the pepStatusDetails block

### ISSUE THREAD

**Issue by ScatteredInk** [openownership/data-standard#345]
This will require validation so that the object can't be empty.

This change is so that we have the same pattern as for listed companies.

**Comment by kd-ods** [comment]
`hasPepStatus` block is currently:

```
    "hasPepStatus": {
      "title": "Politically Exposed Person Status",
      "description": "Is the person described in this statement a politically exposed person? This field should not be used if PEP declarations are not expected as part of this disclosure. If a PEP declaration is expected but missing this field should not be used but the reason for the missing data declared in the pepStatusDetails field.",
      "type": "boolean",
      "propertyOrder": 79
    }
```

It sits alongside `pepStatusDetails`, which is currently an array:

```
    "pepStatusDetails": {
      "title": "Politically Exposed Person Status Details",
      "description": "One or more descriptions of this person's Politically-Exposed Person (PEP) status.",
      "type": "array",
      "items": {
        "$ref": "components.json#/definitions/PepStatusDetails"
      },
      "propertyOrder": 80
    }
```

I suggest:

- rename`pepStatusDetails` to `pepStatus` and make it an object.
- add a new `details` property at the top level of `pepStatus`, making this what is currently the `pepStatusDetails` array (title, description and all)
- move `hasPepStatus` into the `pepStatus` block at the top level, alongside the `details` block.
- make `hasPepStatus` a required field
- edit the `hasPepStatus` description to be "`true` or `false` according to whether the person described by this statement has the status of politically exposed person. A `null` value means a PEP status declaration is expected but missing; the reason for the missing data SHOULD be supplied in the `details` field." 
- title and description for `pepStatus`:

title: Politically Exposed Person status
description: Information about whether, and how, the person described by this statement is politically exposed. This field should not be used if PEP declarations are not expected as part of beneficial ownership declarations.

After this is all done, the relevant test data will need to be updated too.

**Comment by kd-ods** [comment]
Closed by PR #348
